static/app/views/issueDetails/streamline/solutionsSection.tsx

```diff
@@ -49,36 +62,74 @@ export default function SolutionsSection({
     });
   };
 
-  const {data, isPending, hasGenAIConsent} = useGroupSummary(
-    group.id,
-    group.issueCategory
-  );
+  const hasGenAIConsent = organization.genAIConsent;
+  const {data, isPending} = useGroupSummary(group.id, group.issueCategory);
```

Comment:
```suggestion
  const {data: summaryData, isPending: isSummaryPending} = useGroupSummary(group.id, group.issueCategory);
```
Because this component contains more functionality than just group summary, good practice to name these